compactor2/src/components/scratchpad/mod.rs

```diff
@@ -16,6 +16,34 @@ pub trait ScratchpadGen: Debug + Display + Send + Sync {
 }
 
 #[async_trait]
+/// An intermediate in-memory store (can be a disk later if we want)
+/// to stage all inputs and outputs of the compaction. The reasons
+/// are:
+///
+/// *fewer IO ops*: DataFusion's streaming IO requires slightly more IO
```

Comment:
```suggestion
/// **fewer IO ops:** DataFusion's streaming IO requires slightly more IO
```

`*` in proper non-GitHub markdown is italic, while `**` is bold.